index.d/navidshaikh.yaml

```diff
@@ -22,3 +22,15 @@ Projects:
     notify-email: shaikhnavid14@gmail.com
     depends-on: centos/centos:latest
     build-context: ../../
+
+  - id: 3
+    app-id: navidshaikh
+    job-id: ccp-openshift-slave
+    git-url: https://github.com/navidshaikh/container-pipeline-service
+    git-branch: new-scanners
```

Comment:
Typo in git branch https://github.com/navidshaikh/container-pipeline-service/tree/new-scaners your actual branch is single n